apps/www/_blog/2024-08-28-supabase-vercel-partnership.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: 'Supabase + Vercel Partnership'
+description: "Vercel just added official First-Party Integrations. We're one of them."
+author: kamil_ogorek,jonny
+image: vercel-supabase/vercel-supa-thumb.png
+thumb: vercel-supabase/vercel-supa-thumb.png
+categories:
+  - product
+tags:
+  - product
+date: '2024-08-28'
+toc_depth: 3
+---
+
+Vercel just added official [First-Party Integrations](https://vercel.com/blog/share/introducing-the-vercel-marketplace). We're one of them.
+
+This makes it _a lot_ easier to launch Postgres databases from Vercel with full support for [Vercel Templates](https://vercel.com/templates/next.js/supabase) and integrated billing.
+
+<Quote img="lee-robinson.png" caption="Lee Robinson, VP of Product @ Vercel">
+
+Postgres is my favorite database—and Supabase makes Postgres _incredibly_ easy. Plus, having first-party solutions for auth and vector search is so convenient when I'm trying to ship quickly. Now, I can have my Supabase database and my Vercel frontend together in one bill.
+
+</Quote>
+
+## What is the integration?
+
+This integration makes means that you can manage all your Supabase services directly from the Vercel dashboard. You can create, manage, and delete databases and all the credentials are automatically injected into your Vercel environment.
+
+All the billing is unified in your Vercel bill.
+
+## Pairing Vercel & Supabase
+
+Vercel + Supabase have a similar DNA - we're focused on making developers more productive, without compromising on performance & scale. Vercel and Supabase are #1 and #2 most popular for [developers shout outs](https://www.producthunt.com/shoutouts/engineering-development) on ProductHunt. Developers ship faster with Vercel and Supabase.
+
+{/* ![image.png](https://prod-files-secure.s3.us-west-2.amazonaws.com/165a3c21-8e38-4d4b-831a-f9bb88a3262b/63431075-1bed-453a-8670-c666660c92ec/image.png) */}
+
+![Vercel and Supabase on Product Hunt](/images/blog/vercel-supabase/producthunt-vercel-supabase.png)
+
+We've found that Supabase and Vercel has been a very popular pairing for scale ups, YC companies, and large enterprises.
+
+<Quote img="zeno-rocha.png" caption="Zeno Rocha, CEO @ Resend">
+
+I don't think we would have grown so quikcly without Supabase and Vercel. We have used many different products since we started the company, but Supabase and Vercel are the few services that we still use today. Now, there are 180,000 Resend users sending millions of emails every single day, and even though we outgrew many other products, Supabase and Vercel continue to help scale our company despite our challenges evolving all the time.
```

Comment:
```suggestion
I don't think we would have grown so quickly without Supabase and Vercel. We have used many different products since we started the company, but Supabase and Vercel are the few services that we still use today. Now, there are 180,000 Resend users sending millions of emails every single day, and even though we outgrew many other products, Supabase and Vercel continue to help scale our company despite our challenges evolving all the time.
```